### PR TITLE
feat: add Cardano testnet definitions

### PR DESCRIPTION
**Pull request by vladimirvolek**

### CODE REVIEW

File: common/defs/misc/misc.json [hunk 1]
```diff
@@ -87,6 +87,22 @@
             "Daedalus": "https://daedaluswallet.io"
         }
     },
+    {
+        "name": "Cardano Testnet",
+        "shortcut": "tADA",
+        "slip44": 1815,
```
Comment (prusnak) [inline common/defs/misc/misc.json:93]: Every testnet should use 1 for SLIP44.

```suggestion
        "slip44": 1,
```

**Review by prusnak — CHANGES_REQUESTED** [review]

**Comment by matejcik** [comment]
Why do you need this? Just to display `tADA` on trezor.io/coins ?

**Comment by mroz22** [comment]
> Why do you need this? Just to display `tADA` on trezor.io/coins ?

Connect runs this script https://github.com/trezor/connect/blob/develop/scripts/coins-build.sh in order to generate coins.json file which is later used in connect runtime. My understanding is that without the change proposed here, we won't have cardano testnet in the resulting json file.

**Comment by matejcik** [comment]
there are missing support infos
please add them, see https://github.com/trezor/trezor-common/tree/filter-repo/tools#maintaining-support-status

**Review by matejcik — APPROVED** [review]

**Pull request merged**